### DISCUSSION TITLE
Working with URL fragments

### DISCUSSION THREAD

**Discussion by tomassliz** [pointfreeco/swift-url-routing#28] [Q&A]
Is it possible to work with [URL fragments](https://en.wikipedia.org/wiki/URI_fragment)?

> The fragment identifier introduced by a hash mark # is the optional last part of a URL for a document. It is typically used to identify a portion of that document. The generic syntax is specified in RFC 3986. The hash-mark separator in URIs is not part of the fragment identifier.

Fragments are supported in Apple's [URL](https://developer.apple.com/documentation/foundation/url/1779867-fragment) type.

Examples:
- `http://example.org/foo.html#bar`
- `http://example.org/foo.mp4#t=10,20`

**Comment by robfeldmann** [comment]
This question finally motivated me to submit a PR #29 adding this feature. 😄

**Reply by tomassliz** [reply]
@robfeldmann Wau, thank you! I will watch this PR for sure.
